services/user-service/app/routes/auth.py

```
@@ -0,0 +1,244 @@
+    new_access_token = auth_service.create_access_token(user)
+    new_refresh_token = await auth_service.create_refresh_token(redis_client, user_id, token_device)
+
+    # 재발급 완료 시점에 카운터 증가
+    token_refresh_total.add(1)
+    logger.info("토큰 재발급 완료", user_id=user.id)
+    return TokenResponse(access_token=new_access_token, refresh_token=new_refresh_token)
+
+
+@router.post("/logout", status_code=status.HTTP_204_NO_CONTENT)
+async def logout(
+    credentials: HTTPAuthorizationCredentials = Depends(security),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)